crates/bevy_ui/src/render/mod.rs

```diff
@@ -250,6 +255,31 @@ impl ExtractedUiNodes {
     }
 }
 
+/// A [`RenderGraphNode`] that executes the UI rendering subgraph on the UI
+/// view.
+struct RunUiSubgraphOnUiViewNode;
+
+impl RenderGraphNode for RunUiSubgraphOnUiViewNode {
+    fn run<'w>(
+        &self,
+        graph: &mut RenderGraphContext,
+        _: &mut RenderContext<'w>,
+        world: &'w World,
+    ) -> Result<(), NodeRunError> {
+        // Fetch the UI view.
+        let Some(mut render_views) = world.try_query::<&DefaultCameraView>() else {
+            return Ok(());
+        };
+        let Some(default_camera_view) = render_views.iter(world).next() else {
```

Comment:
The `multiple_windows` example doesn't render the label for the second window. It looks like this just needs to look up the correct view using the `graph.view_entity()`.
```suggestion
        let Ok(default_camera_view) = render_views.get(world, graph.view_entity()) else {
```